frontend/src/features/tabs/components/ImageToImage/ImageToImagePanel.tsx

```diff
@@ -43,10 +45,16 @@ export default function ImageToImagePanel() {
     other: {
       header: 'Other Options',
       feature: Feature.OTHER,
-      content: <OutputOptions />,
+      content: <ImageToImageOutputOptions />,
     },
   };
 
+  const dispatch = useAppDispatch();
+
+  const handleChangeHiresFix = () => dispatch(setHiresFix(false));
+
+  handleChangeHiresFix();
```

Comment:
You want to put this inside a useEffect and check if the activetabname is image to image. Otherwise, it'll retrigger for any  change on the page.

Reply:
TIL what a useEffect is. Thanks! Added - you can tell me if I did it right :)